tests/Oryx.Integration.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -64,7 +64,7 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output, TestTempDirTestFix
                     "-c",
                     buildImageScript
                 },
-                _imageHelper.GetRuntimeImage("dotnetcore", "dynamic"),
+                _imageHelper.GetRuntimeImage("dotnetcore", runtimeVersion),
```

Comment:
Wondering why we want to change it here.

Reply:
Good catch. This change should not be required.